packages/dds/map/src/test/memory/map.spec.ts

```diff
@@ -11,6 +14,24 @@ import {
 import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils/internal";
 
 import { type ISharedMap, SharedMap } from "../../index.js";
+const baselineFilePath = path.join(__dirname, "memory_baseline.json");
+
+// Load previous baseline if exists
+const loadBaselines = (): Record<string, number> => {
+	try {
+		return JSON.parse(fs.readFileSync(baselineFilePath, "utf8")) as Record<string, number>;
+	} catch {
+		return {};
```

Comment:
Add a console error? console.error("Error loading baselines:", error);

Reply:
Based on the function docs, it is okay for the baseline to be missing. Might be worth returning `undefined` in that case though, rather than an empty record.

Reply:
Also, this will fail if the file is missing, but it can also fail for other reasons (malformed file contents, for example). It might be better to first check if the file exists and return early if it doesn't. If it does exist, we probably don't want to eat errors that occur while reading the file.